faithcomesbyhearing/sofria-cli
# FCBHDBP-455 It has add a new input parameter to set the folder name o…

# Description
It has add a new input parameter to set the folder name of the json files. This change is related to dbp-etl PR. So, dbp-etl will send the fileset path to create the folder where the JSON files will be stored.

# NOTE
This PR is related with etl PR:
- https://github.com/faithcomesbyhearing/dbp-etl/pull/72

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-455

## How Do I QA This
You can run the following command and it should create JSON files
```shell
node sofria_mediaId.js ~/files/upload/ENGWEBN2ET /tmp/ ENGWEBN_ET
```
